src/sources/organisationLogos.json

```diff
@@ -0,0 +1,461 @@
+{
+    "02gfc7t72": {
```

Comment:
Mooi! Ben benieuwd of dit allemaal goed gaat met de ROR's, weet niet of ik altijd de juist gebruik.